src/current/cockroachcloud/costs.md

```diff
@@ -231,8 +231,8 @@ Cross-region data transfer includes:
 
 - Data transfer required to support queries that involve lookups on nodes in another region.  
 - CockroachDB replication across nodes that are in different regions.  
-- Data egress from the CockroachDB Cloud cluster via supported [private connectivity](https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster#establish-private-connectivity) services to a private endpoint in another region.  
-- [Managed backup](https://www.cockroachlabs.com/docs/cockroachcloud/backup-and-restore-overview#managed-backups) and [Self-managed backup](https://www.cockroachlabs.com/docs/cockroachcloud/take-and-restore-self-managed-backups) data transfer to another region.  
+- Data egress from the CockroachDB Cloud cluster via supported [private connectivity]({% link cockroachcloud/connect-to-your-cluster.md %}#establish-private-connectivity) services to a private endpoint in another region.  
+- [Managed backup]({% link cockroachcloud/backup-and-restore-overview.md %}#managed-backups) and [Self-managed backup]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) data transfer to another region.  
 - Change Data Capture (changefeed) data transfer to another region.
```

Comment:
I wonder if this is where it could be exploded, then you wouldn't need to add it to each tab in the section below.